spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe 'Visit the home page', type: :feature do
+  before do |test|
+    # a game that has both players
+    FactoryGirl.create(:game, name: 'Full game')
+
+    if test.metadata[:open_game]
+      # a game that has only one player (player1)
+      @game2 = FactoryGirl.create(:game, name: 'First open game', player_2: nil)
+
+      # a game that has only one player (player2)
+      @game3 = FactoryGirl.create(:game, name: 'Second open game', player_1: nil)
+    end
+
+    visit root_path
+  end
```

Comment:
I don't think the `before` is needed here. The metadata[:open_game] is nice. But in this case, we only have 2 tests. So it takes extra time to read/understand this code.

What I would do is. Set `game` up as a let (since it's the only one shared in both tests). Then in each test you can do something like this.

```
game

visit root_path

expect(...
```

Or
```
game
game2 = FactoryGirl.create(:game, name: 'First open game', player_2: nil)
game3 = FactoryGirl.create(:game, name: 'First open game', player_2: nil)

visit root_path

expect(...
```